src/main/resources/org/springframework/integration/aws/config/xml/spring-integration-aws-1.0.xsd

```diff
@@ -65,9 +65,13 @@
 		<xsd:complexType>
 			<xsd:annotation>
 				<xsd:documentation>
-					Defines an outbound S3 Channel Adapter for Uploading files to Amazon S3
+						Defines an outbound S3 Channel Adapter for Uploading files to Amazon S3
 				</xsd:documentation>
 			</xsd:annotation>
+			<xsd:choice minOccurs="0" maxOccurs="2">
+				<xsd:element ref="integration:poller" minOccurs="0" maxOccurs="1"/>
+				<xsd:element name="request-handler-advice-chain" type="integration:handlerAdviceChainType" minOccurs="0" maxOccurs="1"/>
+			</xsd:choice>
```

Comment:
That is correct change which really addresses the JIRA premise.
But I'm always open for more minor useful changes on the place. E.g. here I recommended you to take a look to the
`<xsd:attributeGroup ref="integration:channelAdapterAttributes"/>` for this `<s3-outbound-channel-adapter>` element. 

Let me know if it is a bit complex for you and we can raise a new JIRA for that would be some more practice for you how to do the proper contribution.